boards/arm/efm32pg_stk3402a/Kconfig.defconfig

```diff
@@ -45,4 +45,11 @@ config COUNTER_GECKO_RTCC
 
 endif # COUNTER
 
+if ADC
+
+config ADC_0
+	default y
+
+endif # ADC
+
```

Comment:
This code is not necessary since we enable driver instances based on defines generated from DTS.

Reply:
I think I'm starting to understand now how to exactly enable a driver, thanks! 👍